opentelemetry-csharp-sdk-to-seq/Example.WeatherService/Program.cs

```diff
@@ -3,74 +3,60 @@
 using OpenTelemetry.Logs;
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
+// ReSharper disable ExplicitCallerInfoArgument
 
-ActivitySource ExampleWeatherServiceActivitySource = new("Example Weather Service");
+var exampleActivitySource = new ActivitySource("Example.WeatherService");
 
-try
```

Comment:
Catch block is redundant, since allowing the exception to escape `Main()` will also produce a nonzero exit code, and have the side-effect of printing some useful output to stderr.